packages/cloud-cognitive/src/components/DataSpreadsheet/_storybook-styles.scss

```
@@ -0,0 +1,15 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import './index';
+
+// TODO: add any additional styles used by DataSpreadsheet.stories.js
+
+// Uncomment next line (which must appear last) to test in storybook
+// that the SCSS styles for this component are sufficiently specific
+// to override Carbon whichever order the styles get loaded in.
+//@import 'carbon-components/css/carbon-components.min';
```

This was an error, and a bit of obsolete commentary, in the cli templates, sorry, which I've just fixed in another PR. We should not import ./index here as it duplicates styles in storybook.

```suggestion
// TODO: add any additional styles used by DataSpreadsheet.stories.js

```

---

Removed [here](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/1606/commits/005fa39e3e8fd0862820232c0b9d121e0e612a1e)